Cargo.toml

```diff
@@ -20,6 +20,7 @@ name = "cyclors"
 bincode = "1.3.3"
 derivative = "2.2.0"
 libc = "0.2.67"
+log = "0.4.18"
```

Comment:
Set the same version than in zenoh ([0.4.17](https://github.com/eclipse-zenoh/zenoh/blob/814d968116dce82cf03eb535575f13bbbf417dc3/Cargo.toml#L104)).
Otherwise there might be conflict between the plugin and the router, leading to a crash.

Reply:
Version changed to match Zenoh.